scalalib/src/mill/scalalib/dependency/versions/Version.scala

```diff
@@ -0,0 +1,249 @@
+/*
```

Comment:
Let's get rid of the license headers since we haven't been using them throughout most of the codebase; any code contributed ends up licensed under the top-level license in the repo

Reply:
actually, if the bsd license requires copying the license blurb, then that's probably fine.

Reply:
Yes, it's a requirement of all variants of the BSD license (clause number 2 for the 3-clause variant here).